src/main/fc/config.h

```diff
@@ -36,6 +36,9 @@ PG_DECLARE(pilotConfig_t, pilotConfig);
 typedef struct systemConfig_s {
     uint8_t pidProfileIndex;
     uint8_t activeRateProfile;
+#ifdef USE_OSD_PROFILES
+    uint8_t osdProfileIndex;
```

Comment:
Why is this not in `osdConfig`?

Reply:
Correct, it belongs in osdConfig - now moved there.  Thank you.